portfolio/src/main/webapp/styles/style.css

```diff
@@ -39,23 +57,23 @@ h1 {
   position: fixed;
   top: 0;
   width: 100vw;
-  z-index:2;
+  z-index: 2;
 }
 
 /* Left flex element of 'outer-nav' container */
 #home-logo {
   background: rgb(40,40,60);
   border: solid;
-  border-color: rgb(40,40,250,.5);
+  border-color: rgb(6, 6, 126);
   border-radius: 50%;
   box-shadow: 5px 5px 20px rgb(40,40,250,.5);
-  color: #008BB3;
+  color: #008bb3;
```

Comment:
Update to be consistent with the rest of the file using the rgb() syntax. If you have any commonly used colors or numbers, prefer using CSS variables: https://www.w3schools.com/css/css3_variables.asp

Reply:
Ahh I did not know these existed, thanks! Updated.